# Exception

print(X)  #NameError: name 'X' is not defined